modules/s2i/bash/artifacts/usr/local/s2i/assemble

```diff
@@ -54,3 +59,10 @@ if [ "$S2I_ENABLE_JLINK" = "true" ]; then
     }
 
 fi
+
+if [ "$S2I_DELETE_SOURCE" == "true" ]; then
```

Comment:
I'd like this if-block moved into the previous one (which gates all the jlink code around S2I_ENABLE_JLINK)